synapse/lib/types.py

```diff
@@ -553,6 +553,10 @@ def _norm_list(self, valu, oldval=None):
             'xref': pvval,
         }
 
+        for k, v in vsub.items():
```

Comment:
What does this change?

Reply:
When we do normalization of the field that a xref sources from, this adds the subs to the output of the xref.  For example, if `inet:web:postref` was normalized, the XrefType would kick out the subs for any normalization it did for the post.  This is useful if the post is passed in directly as the base components, instead of the guid primary property of the post.  The autoadd system would then be able to pick out any non-model but still-form properties for that post and ensure they are created.

Reply:
This is also in line with "pass the subs up and let the model decide what to do with them" concept.

Reply:
OK makes sense.